client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts

```diff
@@ -158,9 +158,9 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
         let interactedContacts: InteractedContact[] = [];
         interactedContactsLogger.info(`launching server request with epidemiology number ${epidemiologyNumber}`, Severity.LOW);
         axios.get('/contactedPeople/allContacts/' + epidemiologyNumber).then((result: any) => {
-            if (result?.data?.data?.allContactedPeople?.nodes) {
+            if (result?.data && result.headers['content-type'].includes('application/json')) {
```

Comment:
Why do you need to check the content type?

Reply:
Because sometimes the data comes as xml and i want to check that not the case